Detections/ASimWebSession/SuspiciousUAHackTool.yaml

```diff
@@ -0,0 +1,88 @@
+id: 3f0c20d5-6228-48ef-92f3-9ff7822c1954
+name: Suspicious User-Agent Detected - Hacking Tool
+description: |
+  'Detects suspicious user agent strings used by known hack tools
+  To use this analytics rule, make sure you have deployed the [ASIM normalization parsers](https://aka.ms/AdvancedSentinelInformationModel)''
+severity: Medium
+tags:
+  - Id: 8d537f3c-094f-430c-a588-8a87da36ee3a
+    version: 1.0.0
+  - Schema: ASimWebSession
+    SchemaVersion: 0.2.1
+requiredDataConnectors: []
+queryFrequency: 15m
+queryPeriod: 15m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandandControl
+query: |
+  let timeframe = 15m;
+  let user_agents=dynamic([
```

Comment:
Update the list from here: https://github.com/SigmaHQ/sigma/blob/master/rules/proxy/proxy_ua_hacktool.yml (if different)